tests/test_dse.py

```diff
@@ -43,6 +43,25 @@ def test_scheduling_after_rewrite():
     assert trees[1][0].dim == trees[2][0].dim == trees[3][0].dim == grid.time_dim
 
 
+@pytest.mark.parametrize("nx,ny", [(5, 6), (4, 16), (20, 20), (100, 100)])
+def test_skew_vs_advanced(nx, ny):
+    """Trivial testing for DSE skewing"""
+    nx = 10
+    ny = 10
+    timesteps = 1
+    grid = Grid(shape=(nx, ny))
+    u_skew = TimeFunction(name='u_skew', grid=grid)
+    u = TimeFunction(name='u', grid=grid)
+
+    eq_skew = Eq(u_skew.forward, u_skew.dx + u_skew.dy + u_skew.dy.dy)
```

Comment:
reminder: we've discussed about how to simplify this code

Reply:
I did it as presented in #time-tiling thread in Slack.